test/gemm-microkernel-tester.h

```diff
@@ -491,14 +510,29 @@ class GemmMicrokernelTester {
   bool relu_{false};
 };
 
+enum class LoopStepType {
+  Linear,
+  NextPrime
+};
+
 struct LoopParams {
   LoopParams() = default;
-  explicit LoopParams(size_t from, size_t to, size_t step)
-      : is_set(true), from(from), to(to), step(step) {}
+  explicit LoopParams(size_t from, size_t to, size_t step, LoopStepType step_type)
```

Comment:
If preferred, I could change this to a boolean step_prime variable or similar. I figured the enum class approach is slightly more readable, but does feel less idiomatic to the codebase. Feel free to weigh in on this if you prefer it differently.